src/ucs/Makefile.am

```diff
@@ -75,6 +75,7 @@ nobase_dist_libucs_la_HEADERS = \
 	type/param.h \
 	type/init_once.h \
 	type/spinlock.h \
+	type/rwlock.h \
```

Comment:
can we move "introducing fast rwlock to ucs" to separate PR, including gtest for functionality and performance measurement (print lock overhead using UCS_TEST_MESSAGE + write in the PR desc how fast is new lock compared to glibc)